src/NServiceBus.Core/Transports/TransportOperations.cs

```diff
@@ -51,11 +51,11 @@ public TransportOperations(params TransportOperation[] transportOperations)
         /// <summary>
         /// A list of multicast transport operations.
         /// </summary>
-        public IEnumerable<MulticastTransportOperation> MulticastTransportOperations { get; }
+        public List<MulticastTransportOperation> MulticastTransportOperations { get; }
```

Comment:
I think returning arrays would be nicer from an API perspective for scenarios where we know the list is "readonly" but  I'm ok too with lists since this is not on the basic API and it would probably mean additional allocations.